support-frontend/ts-error-history.csv

```diff
@@ -35,4 +35,8 @@
 1637070377000,1135,596e3e86b5e80d3349c6e611a4cb9d7d895a4592
 1637080641000,1116,6a596404e0e8259143074451cbd1381aec01f641
 1637139236000,1116,e92ec73f53bb872c9cd5080884b0a7faa6f63681
+<<<<<<< HEAD
 1637151796000,1116,HEAD
+=======
+1637161943000,1142,HEAD
+>>>>>>> 5e2c7d0f6 (fix TS and ESlint errors)
```

Comment:
I think we'll want to fix this one!

Reply:
Huh, though I'd rebased that. Fixed it now.